server/channels/app/authorization.go

```diff
@@ -299,17 +298,22 @@ func (a *App) HasPermissionToChannel(c request.CTX, askingUserId string, channel
 		return false
 	}
 
-	channelMember, err := a.GetChannelMember(c, channelID, askingUserId)
```

Comment:
I there a reason we shouldn't cache the single member call instead? It just seems counter intuitive to fetch more data than needed purely because we know it's usually cached.

Reply:
It's because adding a new cache is fraught with danger and new bugs. There's just way too many edge cases. I'd rather remove this than add a new cache.

Reply:
Fair enough, I'll diligently wait for Redis to save us all :)